web/themes/custom/cloudy/templates/field/field.html.twig

```diff
@@ -65,24 +65,30 @@
   ])
 %}
 
-{% spaceless %}
+{% apply spaceless %}
   {% block field %}
   <div{{ attributes.addClass(field_classes).addClass(classes) }}>
     {% block label %}
       {% if label_display == "above" %}
         <h2{{ title_attributes.addClass(title_classes) }}>{{ label }}</h2>
-      {% elseif not label_hidden %}
+      {% elseif label_display != "hidden" %}
         <div{{ title_attributes.addClass(title_classes) }}>{{ label }}</div>
       {%- endif -%}
```

Comment:
You can omit the hyphens as spaceless should remove any whitespace.

Reply:
Is that what those hyphens do?! I had wondered 😄